soc/arm/Kconfig

```diff
@@ -13,3 +13,9 @@ config CPU_HAS_NXP_MPU
 	help
 	  This option is enabled when the CPU has a Memory Protection Unit (MPU)
 	  in NXP flavour.
+
+config CPU_HAS_ARM_SAU
+	bool "ARM Security Attribution Unit"
```

Comment:
Is this supposed to be user-configurable (have a prompt)? Feels like hardware information.

Could it come from DTS or a `select` from some higher-level symbol instead?

Reply:
It would come from SOC definition.

Reply:
Ah, I believe, I should remove the prompt, good catch.

Reply:
Yeah, a good rule for whether to add a prompt is "would it makes sense for someone to toggle this in the `menuconfig` interface?"

Reply:
Can consider just this one as the -1, if the prompt was added by accident. :)

Reply:
Removed prompt.